### PR TITLE
docs: provider faq answer for configurable maintenance times for cloud

### PR DESCRIPTION
**Pull request by stevenGravy**
ONly backport to `branch/v12` since it's cloud.

### CODE REVIEW

**Review by bernardjkim — APPROVED** [review]

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 1]
```diff
@@ -25,6 +25,17 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
 
 Please [reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
+## Are maintenance update times configurable? 
+
+Yes, Teleport provides scheduled maintenance windows for upgrades, patches and other required updates. You can select the
+maintenance start window time of day that works best for your organization.
+
+To select the best maintenance time window select your username in the Web UI, then the Help & Support option from the drop-down.
+The Schedule Upgrade section within that page allows you to configure the start time. Updates for that tenant will not start
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:34]:
```suggestion
The Scheduled Upgrade section within that page allows you to configure the start time. Updates for that tenant will not start 
```

Is it schedule or scheduled? I thought it was scheduled.

Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-cloud/faq.mdx]: Thanks I've matched it @zmb3

Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-cloud/faq.mdx]: image

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 2]
```diff
@@ -25,6 +25,17 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
 
 Please [reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
+## Are maintenance update times configurable?
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:28]: We're referring to the same thing with different verbiage a few places here.

- maintenance update times
- scheduled maintenance windows
- maintenance start window time
- maintenance time window

Is there any simplification we could do here?

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 3]
```diff
@@ -25,6 +25,17 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
...
+## Are upgrades times configurable for my Teleport Enterprise Cloud tenant?
+
+Yes, you can set the window start time that works best for your organization for upgrades. Teleport provides a scheduled maintenance
+time window for any upgrades, patches and other required updates for all tenants. An upgrade for your tenant will not start
+earlier than the window start time you set.
+
+To set the window start time select your username in the Web UI, then the Help & Support option from the drop-down.
+The Scheduled Upgrades section within that page allows you to configure the window start time.
+
+Teleport recommends subscribing to the Teleport Enterprise Cloud updates at [status.teleport.sh](https://status.teleport.sh) to keep up to date on scheduled maintenance.
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:37]: Line breaks at 80 chars.

```suggestion
Yes, you can set the window start time that works best for your organization
for upgrades. Teleport provides a scheduled maintenance time window for any
upgrades, patches and other required updates for all tenants. An upgrade for
your tenant will not start earlier than the window start time you set.

To set the window start time select your username in the Web UI, then the Help
& Support option from the drop-down. The Scheduled Upgrades section within that
page allows you to configure the window start time.

Teleport recommends subscribing to the Teleport Enterprise Cloud updates at
[status.teleport.sh](https://status.teleport.sh) to keep up to date on
scheduled maintenance.
```

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**